cargo-guppy/src/main.rs

```diff
@@ -26,6 +27,37 @@ enum Command {
     #[structopt(name = "dups")]
     /// Print the number of duplicate packages
     Duplicates,
+    #[structopt(name = "select")]
+    /// Select packages and their transitive dependencies
+    Select {
+        #[structopt(long, short, possible_values = &Kind::variants(), case_insensitive = true, default_value = "all")]
+        /// Kind of crates to select
+        kind: Kind,
+        #[structopt(long, rename_all = "kebab-case")]
+        /// Include dev dependencies
+        include_dev: bool,
+        #[structopt(long, rename_all = "kebab-case")]
+        /// Include build dependencies
+        include_build: bool,
```

Comment:
Should the default for this be `true`? (If you're unsure could you add a TODO?)

Reply:
My use cases have generally been that I only want the run time dependencies for the actual target. For that reason, I tried to make the defaults match that. `--target` is harder since the only reasonable default is either all targets or the target I'm running on. I chose the former, since the latter is not actually what I want sometimes.

Do you think we should default to the most expansive set and then require people to cull it down? I don't think I have a strong opinion here, so happy to hear your input.